docs/DeveloperGuide.md

```diff
@@ -366,6 +383,19 @@ The following activity diagram summarizes what happens when a user executes a ed
 
 ![Edit command activity diagram](images/EditCommandActivityDiagram.png)
 
+--------------------------------------------------------------------------------------------------------------------
+## **Future iterations**
+
+For future iterations we plan to implement 2 new features.
+
+1. Sending of email to the applicants.
+This will help applicant be notified of their result more quickly.
+In this current iteration we still have to message the client manually.
+
+2. Exporting of Json file into Excel.
+The university admin staff might want to process or analyse the result in a more sophisticated manner.
+Excel is needed to aid in more sophisticated analysis as it has more functions.
+
```

Comment:
rather than Future iterations write it like 
**Proposed features** - Coming soon in future iterations
Or smth like that

Reply:
Ok done